archivist/parser.py

```diff
@@ -88,6 +94,15 @@ def common_parser(description):
         required=True,
         help="FILE containing API authentication token",
     )
+    parser.add_argument(
+        "-n",
+        "--namespace",
+        type=str,
+        dest="namespace",
+        action="store",
+        default=None,
+        help="namespace of item population",
+    )
```

Comment:
This argument already exists in various samples (Wavestone, for example) in the `archivist_samples` repo which already use `common_parser`. Won't this cause errors if we add in both places? Should have an Issue to remove namespace from the individual samples and investigate whether there's any knock-on fixes to be made in the sample code to use the standard namespace

Reply:
archivist-samples currently uses a copy of this so no problems enviaged.